src/pubsub/peer-streams.js

```diff
@@ -107,22 +109,23 @@ class PeerStreams extends EventEmitter {
       throw new Error('No writable connection to ' + id)
     }
 
+    // @ts-ignore - this.outboundStream could be null
     this.outboundStream.push(data)
   }
 
   /**
    * Attach a raw inbound stream and setup a read stream
    *
-   * @param {DuplexIterableStream} stream
+   * @param {MuxedStream} stream
    * @returns {void}
    */
   attachInboundStream (stream) {
     // Create and attach a new inbound stream
     // The inbound stream is:
     // - abortable, set to only return on abort, rather than throw
     // - transformed with length-prefix transform
-    this._inboundAbortController = new AbortController()
     this._rawInboundStream = stream
+    // @ts-ignore - abortable returns AsyncIterable and not a MuxedStream
```

Comment:
this can be removed now that MuxedStream extends AsyncIterable

Reply:
In fact, the abort receives the MuxedStream and returns a AsyncIterable<any>. So, as `this.inboundStream` is always wrapped with the abort I changed it from `MuxedStream` to `AsyncIterable<Uint8Array>`